0/bump_version.py

```diff
@@ -19,6 +19,9 @@ def update_version_in_file(new_version: str):
 
 
 def bump_version():
+    if os.getenv('PLUGIN_SKIP_COMMIT_AUTHOR') == os.getenv("DRONE_COMMIT_AUTHOR"):
+        return
```

Comment:
Should we print a warning / info message here ?

Reply:
done